utils/bazel/overlay_directories.py

```diff
@@ -86,12 +86,15 @@ def main(args):
                 os.path.join(args.overlay, relpath), os.path.join(args.target, relpath)
             )
 
-        for src_entry in os.listdir(os.path.join(args.src, rel_root)):
-            if src_entry not in dirs:
-                relpath = os.path.join(rel_root, src_entry)
-                _symlink_abs(
-                    os.path.join(args.src, relpath), os.path.join(args.target, relpath)
-                )
+        src_path = os.path.join(args.src, rel_root)
```

Comment:
Note: This just makes it so that new directories (like the `platform` directory) that aren't present in the original sources don't raise errors. I pulled this out to #126759

Reply:
Fixed the bug that caused CI issues (https://github.com/llvm/llvm-project/commit/55ae118db7c36fb88994e072e424cb66d1f34e74). No need to pull this out.

Reply:
You might want to write some comments since this is getting complex